### ISSUE TITLE
A C function pointer can only be formed from a reference to a 'func' or a literal closure

### ISSUE THREAD

**Issue by giandifra** [Dev-hwang/flutter_foreground_task#278]
Hello, I've this error on `AppDelegate.swift`:

![Screenshot 2024-09-23 alle 12 41 39](https://github.com/user-attachments/assets/70a5cc23-4386-4d29-a1cc-30a6e6dfb150)

**Comment by Dev-hwang** [comment]
@giandifra 

hello, try this

```swift
SwiftFlutterForegroundTaskPlugin.setPluginRegistrantCallback { registry in
   GeneratedPluginRegistrant.register(with: registry)
}
if #available(iOS 10.0, *) {
   UNUserNotificationCenter.current().delegate = self
}
```

**Comment by giandifra** [comment]
It's working, thanks!
